cmd/telemetrygen/internal/traces/worker.go

```diff
@@ -43,7 +43,9 @@ func (w worker) simulateTraces() {
 			attribute.String("span.kind", "client"), // is there a semantic convention for this?
```

Comment:
This line should be removed

Reply:
You are completely right and I have no idea how I missed that. I have made the changes.